lib/matplotlib/axes/_axes.py

```diff
@@ -4695,34 +4696,35 @@ def fill(self, *args, **kwargs):
 
         Parameters
         ----------
-        args : a variable length argument
-            It allowing for multiple
-            *x*, *y* pairs with an optional color format string; see
-            :func:`~matplotlib.pyplot.plot` for details on the argument
-            parsing.  For example, each of the following is legal::
+        args : sequence of x, y, [color]
+            Each polygon is defined by the lists of *x* and *y* positions of
+            its nodes, optionally followed by by a *color* specifier. See
+            :mod:`matplotlib.colors` for supported color specifiers. The
+            standard color cycle is used for polygons without a color
```

Comment:
iirc this is the same cycle, but a different cycler instance than the one used by `plot`.  Not sure if that is a detail worth including here.

Reply:
Is this important and intentional? Do we want to guarantee this as part of the API?

If not, I would consider it an implementation detail and not document it.